test/test_composite.py

```diff
@@ -186,11 +186,11 @@ def test_analysis_replace_results_true(self):
 
         # Additional data not part of composite experiment
         exp3 = FakeExperiment([0, 1])
-        extra_data = exp3.run(FakeBackend())
+        extra_data = exp3.run(FakeBackend()).block_for_results()
```

Comment:
Is this relevant to copy?

Reply:
This was just to be safe that the child fake experiment finishes all its analysis before later re-running analysis with replacement on the data.